src/libs/nav2_util/src/sensors/amcl_sensor.cpp

```diff
@@ -45,23 +45,23 @@ AMCLSensor::~AMCLSensor()
 
 ////////////////////////////////////////////////////////////////////////////////
 // Apply the action model
-bool AMCLSensor::UpdateAction(pf_t *pf, AMCLSensorData *data)
+bool AMCLSensor::UpdateAction(pf_t * /*pf*/, AMCLSensorData * /*data*/)
```

Comment:
Why do we have all these functions that don't do anything? Is this really an abstract base class? Could we make these pure virtual instead?

Reply:
At this stage I don't want to change the code.  Once, I start changing the code we can address this issue.